### ISSUE TITLE
Troxel v1 roadmap

### ISSUE THREAD

**Issue by chrmoritz** [chrmoritz/Troxel#31]

**Comment by chrmoritz** [comment]
[IgnitasTroveMinimap.zip](https://github.com/chrmoritz/Troxel/files/66341/IgnitasTroveMinimap.zip)

**Comment by chrmoritz** [comment]
* TypeScript
* Angular2
* application cache => Service Workers
* local storage => indexedDB

**Comment by ChtiClem** [comment]
Hello,
I've a question. I'm using Troxel to verify my gun's mesh in order to submit it in the trove creation's reddit.
I've defined the 4 files : original, the type, the alpha and the specular.
In zoxel, it's working fine.
In Trove it's fine too.
But in Troxel, It warns me that I've put some alpha voxel on non existing one and some specular the same (and proposes me a correction).
The problem, is that the fix removes some glass voxels and fix some specular voxels (can't see any difference for this)
I've checked numerous times (and my gun doesn't have so much voxels). I've redone the t, a and s files but Troxel still warns me with the alpha and specular problems.
I provide you my files, just if you can check by yourself if you have the same message, please ?
Or to prove I'm completly dumb.
Thanks in advance (in both cases, ;) )

[BugFiles.zip](https://github.com/chrmoritz/Troxel/files/120512/BugFiles.zip)
[ErrorsLog.txt](https://github.com/chrmoritz/Troxel/files/120513/ErrorsLog.txt)

**Comment by chrmoritz** [comment]
You're missing the most important line in your logs:
`z-axis orientation: 0 (0 for left, 1 for right handed (recommended))`

Your main material map file (`TwinsV3.qb`) is exported with the **right** handed z-axis orientation format, but all other material maps (`_a`, `_s` and `_t`) are exported with the **left** handed z-axis orientation format, which is quite deprecated. Troxel's Qubicle Importer gets confused by your mix of  z-axis orientation formats (it assumes that all files are in the same format).

Currently Zoxel is the only editor (which I know) using the left handed z-axis orientation format. I've opened chrmoritz/zoxel#12 to migrate Zoxel to use the right handed format too.

In summary the issue is caused by using multiple editors for the different material map files. To fix is just open the main file in Zoxel and save it in the old left handed format to be consistent with all other material maps (or convert all 3 material maps one by one to right handed using Troxel).

**Comment by chrmoritz** [comment]
This is fixed inside Troxel with https://github.com/chrmoritz/Troxel/commit/613aa50118d0dfdfb85e4995abb6af3ad4ccae54 now by throwing an error if trying to mix different z-axis orientation formats. I don't think it's worth backporting to the old site. It will most likely released with the migration to the new website. Thanks for reporting this.

**Comment by chrmoritz** [comment]
@ChtiClem The issues was fixed in Zoxel with https://github.com/chrmoritz/zoxel/commit/c570967e580a428541164d521c4192e200b3e925 by using the right handed z-axis orientation format when exporting models to `.qb`(Qubicle). A new version 0.6.1 with the fix was released at https://github.com/chrmoritz/zoxel/releases/tag/0.6.1. Try to update to this version and rexport all material maps.

**Comment by ChtiClem** [comment]
Thanks to you ^_^ I'll give a try either option : or in zoxel or in troxel.
